tests/actor/test_mock_state_manager.py

```diff
@@ -0,0 +1,116 @@
+import unittest
+
+from dapr.actor import Actor, ActorInterface
+from dapr.actor.runtime.mock_actor import create_mock_actor
+from dapr.actor.runtime.mock_state_manager import MockStateManager
+
+
+def double(_: str, x: int) -> int:
+    return 2 * x
+
+
+class MockTestActorInterface(ActorInterface):
+    pass
+
+
+class MockTestActor(Actor, MockTestActorInterface):
+    def __init__(self, ctx, actor_id):
+        super().__init__(ctx, actor_id)
+
+
+class ActorMockActorTests(unittest.IsolatedAsyncioTestCase):
+    def test_init_state(self):
+        mock_actor = create_mock_actor(MockTestActor, 'test')
+        state_manager = mock_actor._state_manager
+        self.assertIsInstance(state_manager, MockStateManager)
+        self.assertFalse(state_manager._mock_state)
+        self.assertFalse(state_manager._mock_reminders)
+        self.assertFalse(state_manager._mock_timers)
+
+    async def test_add_state(self):
```

Comment:
I think we can merge most of these tests (the add, get, set, remove, contains...) into one single test, so we don't have a lot of repetition. Just to keep the tests snappier and infra usage lower.